backend/core/users/verification.py

```
@@ -0,0 +1,60 @@
+	
+	return False
+
+
+def clear_code(user_id: int) -> None:
+	"""Clear verification code for a user.
+	
+	Args:
+		user_id: User ID to clear code for.
+	"""
+	if user_id in _verification_codes:
+		del _verification_codes[user_id]
```

Ruff (0.15.4)

[warning] Use `pop` instead of `key in dict` followed by `del dict[key]`

Replace `if` statement with `.pop(..., None)`

(RUF051)